modules/tinymce/src/plugins/autosave/test/ts/browser/AutoSavePluginTest.ts

```diff
@@ -12,78 +12,83 @@ UnitTest.asynctest('browser.tinymce.plugins.autosave.AutoSavePluginTest', (succe
   Plugin();
   Theme();
 
-  suite.test('TestCase-TBA: AutoSave: isEmpty true', function (editor) {
-    LegacyUnit.equal(editor.plugins.autosave.isEmpty(''), true);
-    LegacyUnit.equal(editor.plugins.autosave.isEmpty('   '), true);
-    LegacyUnit.equal(editor.plugins.autosave.isEmpty('\t\t\t'), true);
+  const checkIfEmpty = (editor: Editor, html: string, isEmpty: boolean): void => {
+    Assert.eq('The HTML "' + html + '" is ' + isEmpty ? 'empty.' : 'not empty.', editor.plugins.autosave.isEmpty(html), isEmpty);
```

Comment:
This ternary will not work as expected, as it'll evaluate as one of the following:
- `'The HTML "' + html + '" is empty'`
- `'not empty'`

Reply:
This also looks to be why the DeepScan checks are failing.